lambda/translcore.ml

```diff
@@ -333,7 +334,7 @@ and transl_exp0 ~in_new_scope ~scopes e =
       end else begin match cstr.cstr_tag with
         Cstr_constant n ->
           Lconst(const_int n)
-      | Cstr_unboxed ->
+      | Cstr_unboxed _ ->
```

Comment:
Was this not forgotten in older commits that changed the type of `Cstr_unboxed`?

Reply:
This bit of code looks correct as far as I know (we don't need the head_shape here).

Reply:
If I understand correctly, you had a first commit that added arguments to the `Cstr_unboxed` constructor, and then this change came in a later commit, when it should have been part of this first commit. (Maybe I was mistaken when I made this remark, because the code of the first commit would not compile if this was true.)

Reply:
It is indeed introduced in the first commit of the branch